adafruit_dps310/basic.py

```diff
@@ -231,43 +237,43 @@ def altitude(self):
         )
 
     @property
-    def temperature(self):
+    def temperature(self) -> float:
         """The current temperature reading in degrees Celsius"""
         _scaled_rawtemp = self._raw_temperature / self._temp_scale
         _temperature = _scaled_rawtemp * self._c1 + self._c0 / 2.0
         return _temperature
 
     @property
-    def sea_level_pressure(self):
+    def sea_level_pressure(self) -> int:
```

Comment:
The getter and setter for `sea_level_pressure` can actually be a `float`, and it being initialized to one in `__init__()`.

Reply:
sorry.  missed that in the initialize....